parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -154,12 +155,12 @@ logs_to_events "AKS.CSE.installNetworkPlugin" installNetworkPlugin
 if [ "${IS_KRUSTLET}" == "true" ]; then
     versionsWasm=$(jq -r '.Packages[] | select(.name == "containerd-wasm-shims") | .downloadURIs.default.current.versionsV2[].latestVersion' "$COMPONENTS_FILEPATH")
     downloadLocationWasm=$(jq -r '.Packages[] | select(.name == "containerd-wasm-shims") | .downloadLocation' "$COMPONENTS_FILEPATH")
-    downloadURLWasm=$(jq -r '.Packages[] | select(.name == "containerd-wasm-shims") | .downloadURIs.default.current.downloadURL' "$COMPONENTS_FILEPATH")
+    downloadURLWasm=$(jq -r '.Packages[] | select(.name == "containerd-wasm-shims") | .downloadURIs.default.current.packagesURL' "$COMPONENTS_FILEPATH")
```

Comment:
still not sure why we are using a new property in the schema ? I dont see a reason why we need to keep using .downloadURL = acs-mirror